visual-python/src/saucelabs_visual/regions.py

```diff
@@ -6,22 +6,28 @@ class Region:
     aliases: List[str] = []
     graphql_endpoint: str
     job_url_template: str
+    visual_url_template: str
 
     def __init__(
             self,
             name: str,
             aliases: List[str],
             graphql_endpoint: str,
-            job_url_template: str
+            job_url_template: str,
+            visual_url_template: str,
     ):
         self.name = name
         self.aliases = aliases
         self.graphql_endpoint = graphql_endpoint
         self.job_url_template = job_url_template
+        self.visual_url_template = visual_url_template
 
     def job_url(self, job_id: str) -> str:
         return self.job_url_template.format(JOB_ID=job_id)
 
+    def build_url(self, build_id: str) -> str:
+        return self.visual_url_template.format(BUILD_ID=build_id)
+
```

Comment:
Shouldn't we rely on the `url` field that the API is returning ?

Reply:
Yep, makes sense to me. I'll update it